libs/libvtrutil/src/vtr_cache.h

```diff
@@ -5,19 +5,21 @@
 
```

Comment:
Missing a file description? (Not sure if it is needed in this case, but it is worth keeping consistency with other headers)